charts/caraml-authz/templates/service.yaml

```diff
@@ -1,13 +1,18 @@
+{{- $globServiceName := include "common.get-component-value" (list .Values.global "authz" (list "serviceName")) }}
+{{- $serviceName := include "common.set-value" (list (include "caraml-authz.fullname" .) $globServiceName) }}
+{{- $globServicePort := include "common.get-component-value" (list .Values.global "authz" (list "externalPort")) }}
+{{- $servicePort := include "common.set-value" (list (.Values.deployment.service.port | toString) $globServicePort) | atoi }}
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "caraml-authz.fullname" . }}
+  name: {{ $serviceName }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
is namespace necessary?

Reply:
no, not a necessary field, added it for consistency.

Reply:
there are debates around this https://github.com/helm/helm/issues/5465
and we don't have a consistent approach across our charts yet

Reply:
updated.